src/plcc/load_spec/validate_spec/validate_syntactic_spec/build_parsing_table_test.py

```diff
@@ -0,0 +1,90 @@
+from pytest import raises
+from .build_parsing_table import build_parsing_table, Table
+from collections import defaultdict
+
+def test_empty():
+    table = build_parsing_table({}, {}, {})
+    assert len(table.getFilledCellLocations()) == 0
+
+def test_set_cell():
+    table = build_parsing_table({}, {}, {})
+    table.setCell('nonterminal', 'TERMINAL', ["PRODUCTION"])
+    assert table.getCell('nonterminal', 'TERMINAL') == ["PRODUCTION"]
+
+def test_left_recursive():
```

Comment:
Where does the left recursion appear in the test? And what is its result? Try to encode this in the test name and its implementation. For example (if this is right)...

`test_left_recursion_results_in_multiple_cell_entries`

The body may be more challenging. Start with comments. Then think about ways you could write it that could encode the comment in variable or function names. If it would ruin the structure too much then don't. And just leave the comments.